pages/facts.html

```diff
@@ -4,9 +4,84 @@
+        <ol>
+          <li>
+            <p class="points">Friends wasn't called Friends</p>
+            <p>Originally, different name suggestions included: Six of One, Friends Like Us and Insomnia Café.
+              In the end, they went with the simplest option, and we can all agree its the best!
+              Insomnia Café doesn't really roll off the tongue...
+            </p>
+            <img class="body-image" src="../images/friends.jpg" alt="group of friends">
+          </li>
+          <li>
+            <p class="points">Phoebe's twin was originally on another show</p>
+            <p>You probably remember Urusula as Phoebe's mean twin, who was always messing with her life. But you may not know her origin story!
+              Lisa Kudrow played a waitress called Ursula on another sitcom called Mad About You.
+              Since Lisa was appearing on both shows at the same time, and both were set in New York, the writers decided it would be funny to make Ursula a character on Friends too, as Phoebe's evil twin!</p>
+            <img class="body-image" src="../images/friends2.jpg" alt="two googly eyed girls">
+          </li>
+
+          <li>
+            <p class="points">You can visit Central Perk in real life! </p>
+            <p>Central Perk (Like Central Park, geddit?) is sadly not a real cafe. BUT you can visit replicas of it, in Liverpool and China!</p>
+            <img class="body-image" src="../images/friends3.jpg" alt="smiling tea cup">
+          </li>
+          <li>
+            <p class="points"> Marcel the monkey is a girl</p>
+            <p>She's called Katie and she's a pretty prolific actor, she's also been in 30 Rock among other things. Katie is a capuchin monkey,
+              who are famous for being featured in films and tv because of their clever ability to learn tricks and expressions.
+            </p>
+            <img class="body-image" src="../images/friends4.jpg" alt="monkey">
+          </li>
+          <li>
+            <p class="points">The show was filmed in LA.</p>
+            <p>Yup, not a single shot is filmed in New York! Even though it's arguably one of the most famous sitcoms set in The Big Apple, it was all filmed in Hollywood, California.
+              The only exception is the shots of the apartments from the outside. They're real, and you can go and see them in Manhattan!</p>
+            <img class="body-image" src="../images/friends5.jpg" alt="Hollywood">
+          </li>
+
+        </ol>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <div>
+      <p><img id="logo-sm" src="../images/Friends_logo.svg" alt="Friends Logo"></p>
+      A super cool Friends Fansite, created with ❤️ by
+        <a href="https://github.com/Havana82/" target="_blank">Halimat Aremu</a>,
+        <a href="https://github.com/kguarnizo" target="_blank">Katherine Guarnizo</a>, and
+        <a href="https://github.com/lissetet" target="_blank">Liz Trejo</a>.
+
+    </div>
```

Comment:
Do we need the div here? Since it doesn't convey any semantic meaning, could we omit it altogether?